crates/wasmtime/src/engine/serialization.rs

```diff
@@ -218,6 +218,10 @@ impl Metadata<'_> {
             component_model_values,
             component_model_nested_names,
 
+            // Not yet implemented; we don't have knobs for these.
+            custom_page_sizes: _,
+            shared_everything_threads: _,
```

Comment:
Mind switching these to `assert!(!custom_page_sizes)` like a few other unimplemented proposals below?

(that way this is caught once we add knobs in the future)